controllers/cloud.redhat.com/clowdenvironment_controller.go

```diff
@@ -31,14 +31,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
+	// Import the providers to initialize them
```

Comment:
Could these imports be pushed to `providers.go`? you could perhaps then do the sort in there, too.

Reply:
No, there is a cyclical issue. I wanted the definition of the ProviderAccessor to be inside the ``provider`` file for the provider. In doing this it means that we need to import them where we use them, we can't import them inside the ProviderRegistration, because they themselves manipulate the provider registration. Thinking of another way to do it.